src/pages/index.tsx

```diff
@@ -1,9 +1,10 @@
 import Head from 'next/head'
+import Layout from '../components/layout'
 import styles from '../styles/Home.module.css'
 
-const Home = () => {
+const Home = (): JSX.Element => {
   return (
-    <div className={styles.container}>
+    <Layout>
       <Head>
         <title>ScribeDAO</title>
```

Comment:
We can move `<title>` to Layout to where `ScribeDAO` is default and then if you pass a title prop to `<Layout>` it becomes `PROP_VALUE | ScribeDAO`

Reply:
Sick, done. future state, i think a `siteMetadata` file could make sense for static things like the title and certain copy.